core/templates/components/state-editor/state-editor-properties-services/state-interaction-id.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for the State interaction id service.
+ */
+
+import { TestBed } from '@angular/core/testing';
+
+/* eslint-disable max-len */
+import { StateInteractionIdService } from
+  'components/state-editor/state-editor-properties-services/state-interaction-id.service.ts';
+/* eslint-enable max-len */
+
+describe('State interaction id service', () => {
+  let siis: StateInteractionIdService = null;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [StateInteractionIdService]
+    });
+
+    siis = TestBed.get(StateInteractionIdService);
+  });
+
+  it('should test the setterMethodKey property.', () => {
```

Comment:
I think the it blocks should have a behavior description instead of only testing its implementation
```suggestion
  it('should has a default value in setterMethodKey', () => {
```

Reply:
done